src/main/java/frc/robot/subsystems/LanuchMechanisumSubsystem.java

```diff
@@ -4,24 +4,54 @@
 
 package frc.robot.subsystems;
 
-import edu.wpi.first.wpilibj.motorcontrol.PWMSparkMax;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.REVPhysicsSim;
+
+import edu.wpi.first.math.system.plant.DCMotor;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Constants;
 
 public class LanuchMechanisumSubsystem extends SubsystemBase {
-    PWMSparkMax topLaunch;
-    PWMSparkMax bottomLaunch;
+    CANSparkMax top;
+    CANSparkMax bottom;
+    CANSparkMax staging;
+    CANSparkMax pickup;
 
   /** Creates a new LanuchMechanisumSubsystem. */
   public LanuchMechanisumSubsystem() {
 
-  topLaunch = new PWMSparkMax(Constants.PWM.topLaunch);
-  bottomLaunch = new PWMSparkMax(Constants.PWM.bottomLaunch);
+    top = new CANSparkMax(Constants.CANSparkMaxID.topRight, MotorType.kBrushless);
+    bottom = new CANSparkMax(Constants.CANSparkMaxID.topLeft, MotorType.kBrushless);
+    staging = new CANSparkMax(Constants.CANSparkMaxID.staging, MotorType.kBrushless);
+    pickup = new CANSparkMax(Constants.CANSparkMaxID.launcherLoad, MotorType.kBrushless);
```

Comment:
you may want to reconsider how you have your `Constants.CANSparkMaxID` values named, but I wouldn't consider it a blocker here.

Reply:
#29